tests/operations.go

```diff
@@ -233,6 +239,14 @@ func (ops *Operations) IsPVCDeleted(pvcName string) bool {
 	return false
 }
 
+// GetPVName gives the pv name for the given pvc
+func (ops *Operations) GetPVName(pvcName string) string {
```

Comment:
can we rename this to `GetPVNameFromPVCName`

Reply:
Done